python/paddle/fluid/layers/nn.py

```diff
@@ -12008,3 +12009,114 @@ def _get_default_param_initializer():
 
     output = helper.append_bias_op(pre_bias, dim_start=1, dim_end=2)
     return output
+
+
+def deformable_psroi_pooling(input,
+                             rois,
+                             trans,
+                             no_trans=0,
+                             spatial_scale=1.0,
+                             output_channels=64,
```

Comment:
why the default `output_channels` is 64.

Reply:
change default value to input channels